packages/altair-app/src/app/modules/altair/components/doc-viewer/doc-viewer/doc-viewer.component.scss

```
@@ -0,0 +1,11 @@
+.doc-viewer-search-label {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)